website/sidebarsIntegrations.js

```diff
@@ -97,6 +97,7 @@ module.exports = {
                     label: "Miscellaneous",
                     items: [
                         "services/gravitee/index",
+                        "services/freshrss/index",
```

Comment:
```suggestion
                        "services/freshrss/index",
                        "services/gravitee/index",
```
We're trying to keep this list sorted.

Reply:
Done. Sorry about that, I was looking for alphabetical order, and missed the first one....